playlists/excel.yaml

```diff
@@ -496,10 +496,19 @@
   group: Preview APIs
   api_set:
     ExcelApi: 1.7
+- id: excel-events-tablecollection-changed
+  name: Events - Table collection changed
+  fileName: events-tablecollection-changed.yaml
+  description: Add event handlers for table collection onDataChanged event
+  rawUrl: >-
+    https://raw.githubusercontent.com/<ACCOUNT>/<REPO>/<BRANCH>/samples/excel/89-preview-apis/events-tablecollection-changed.yaml
+  group: Preview APIs
+  api_set:
+    ExcelApi: 1.7
 - id: excel-events-worksheet-activated
   name: Events - Worksheet activated
   fileName: events-worksheet-activated.yaml
-  description: Add an event handler on the worksheet activated and deactivated events
+  description: Add an event handler for worksheet onActivated and onDeactivated events
```

Comment:
For consistency with the wording of the next snippet's description (excel-events-worksheet-changed), suggest you change this description to:

Add event handlers for worksheet onActivated and onDeactivated events